CMakeLists.txt

```diff
@@ -205,6 +207,98 @@ if (NN_ENABLE_NANOCAT)
     target_link_libraries (nanocat ${PROJECT_NAME})
 endif ()
 
+find_program (ASCIIDOCTOR_EXE asciidoctor)
```

Comment:
originally I had a plan to make the default value of using asciidoctor depend on this.  I couldn't quite achieve this with a fatal error, so by checking it here, I can disable it.  arguably I could enclose this in an if branch, but apart from saving the time to check (few dozen microsends of the build time I think), it doesn't buy me anything else.  I prefer to keep it simple as it is for now.